demos/src/Nodes/Mention/React/MentionList.jsx

```diff
@@ -27,7 +27,7 @@ export default forwardRef((props, ref) => {
 
   useEffect(() => setSelectedIndex(0), [props.items])
 
-  useImperativeHandle(ref, () => ({
+  useImperativeHandle(props.ref, () => ({
```

Comment:
Will Tiptap 3 be compatible with react 18 or will it require React 19?

Reply:
I am worried that this change will make it incompatible with React 18, which does not support ref as props.

Also, React 19 still supports `forwardRef` (they plan to deprecate it in the next versions).

https://react.dev/blog/2024/12/05/react-19#ref-as-a-prop